### PR TITLE
[web] blur view web support

### PR DESCRIPTION
**Pull request by EvanBacon**
# Why

Adds support for web blur view

# Test Plan

TBD

### CODE REVIEW

**Comment by EvanBacon** [comment]
## Safari
Screen Shot 2019-04-04 at 5 23 49 PM

## Chrome

Screen Shot 2019-04-04 at 5 24 11 PM

**Comment by EvanBacon** [comment]
Note that this changes the default behavior of Android intensity.

File: packages/expo-blur/src/BlurView.web.tsx [hunk 1]
```diff
@@ -1,28 +1,51 @@
...
-} & React.ComponentProps<typeof View>;
-type BlurTint = 'light' | 'dark' | 'default';
+import { BlurTint, Props } from './BlurView.types';
+import getBackgroundColor from './getBackgroundColor';
 
+function isBlurSupported(): boolean {
+  // https://developer.mozilla.org/en-US/docs/Web/API/CSS/supports
+  // https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter#Browser_compatibility
+  // TODO: Bacon: Chrome blur seems broken natively
+  return CSS.supports('-webkit-backdrop-filter', 'blur(1px)');
```
Comment (ide) [inline packages/expo-blur/src/BlurView.web.tsx]: some browsers don't support CSS.supports:
```suggestion
  return typeof CSS !== 'undefined' && CSS.supports('-webkit-backdrop-filter', 'blur(1px)');
```

File: packages/expo-blur/src/BlurView.web.tsx [hunk 2]
```diff
@@ -1,28 +1,51 @@
...
+import { View, ViewPropTypes } from 'react-native';
 
-type Props = {
-  tint: BlurTint;
-} & React.ComponentProps<typeof View>;
-type BlurTint = 'light' | 'dark' | 'default';
+import { BlurTint, Props } from './BlurView.types';
+import getBackgroundColor from './getBackgroundColor';
 
+function isBlurSupported(): boolean {
```
Comment (ide) [inline packages/expo-blur/src/BlurView.web.tsx]: Move helper functions below the functions/classes that use them --> 

1. code of primary interest
2. implementation details

establishes context better than the other way around

**Review by ide — APPROVED** [review]

**Pull request merged**